src/wui/lwip.c

```diff
@@ -63,9 +63,8 @@
 
```

Comment:
i dont like that we directly access eeproms in wui function. Ok it can be accepted now, but in future the "wui_api.c should have that interface function which gets all settings from EEPROM and set to EEPROM